src/node_crypto.cc

```diff
@@ -3952,6 +3953,41 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment *env, v8::Local<v8::Object> options,
+                            int *padding, int *saltlen) {
+  Local<Value> key = String::NewFromUtf8(env->isolate(), "padding");
+  MaybeLocal<Value> maybePadding = options->Get(key);
+  if (!maybePadding.IsEmpty()) {
+    Local<Value> paddingValue = maybePadding.ToLocalChecked();
+    if (paddingValue->IsString()) {
+      v8::String::Utf8Value paddingUtf8(paddingValue);
+      const char *paddingName = *paddingUtf8;
```

Comment:
Actually… `crypto.constants.RSA_PKCS1_PSS_PADDING` and `crypto.constants.RSA_PKCS1_PADDING` already exist and are accessible from userland. Maybe we could use these constants instead of a string?